assets/wizards/googleAdManager/index.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * Google Ad Manager Wizard.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, render, Fragment } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { withWizard, FormattedHeader, Handoff } from '../../components/src';
```

Comment:
`FormattedHeader` and `Handoff` aren't in the component.

```suggestion
import { withWizard } from '../../components/src';
```